src/pipecat/processors/frameworks/rtvi.py

```diff
@@ -343,6 +350,16 @@ class RTVIBotStoppedSpeakingMessage(BaseModel):
     type: Literal["bot-stopped-speaking"] = "bot-stopped-speaking"
 
 
+class RTVIMetricsMessageData(BaseModel):
+    metrics: dict
+
+
+class RTVIMetricsMessage(BaseModel):
+    label: Literal["rtvi-ai"] = "rtvi-ai"
+    type: Literal["metrics"] = "metrics"
+    data: RTVIMetricsMessageData
```

Comment:
I'm going to flatten this, as you suggested, and just have:
```
class RTVIMetricsMessage(BaseModel):
    label: Literal["rtvi-ai"] = "rtvi-ai"
    type: Literal["metrics"] = "metrics"
    data: Mapping[str, Any]
```

Seems cleaner since now you can just access the metrics type directly instead of data.ttfb, for example.